h2/src/main/org/h2/command/Parser.java

```diff
@@ -698,6 +701,32 @@ private Schema getSchema(String schemaName) {
     private Schema getSchema() {
         return getSchema(schemaName);
     }
+    /*
```

Comment:
@grandinj - I added an explanation in the code instead of reply to your review comment - please read this. (Sorry I have limited internet right now - so this was the best choice on with the few megabytes of internet I had today...)

If you like - I can removed the side effect of setting the new schemaName - but the comments still apply...